mappings/18w15a#18w16a.tinydiff

```diff
@@ -3052,11 +3052,11 @@ c	net/minecraft/unmapped/C_5587144$C_6955547
 		p	4		p_4	
 		p	5		p_5	
 	m	(Lnet/minecraft/unmapped/C_2137190;Lnet/minecraft/unmapped/C_3674802;Lnet/minecraft/unmapped/C_2441996;Lnet/minecraft/unmapped/C_2441996;Lnet/minecraft/unmapped/C_2441996;)Z	m_4699966		canPlaceBlock
-		p	1			p_1
-		p	2			p_2
-		p	3			p_3
-		p	4			p_4
-		p	5			p_5
+		p	1			worldView
```

Comment:
`World` implements various interfaces, and variables of those types can still just be called `world`